### ISSUE TITLE
Multiple stores?

### ISSUE THREAD

**Issue by jjeff** [zoubingwu/electron-shared-state#5]
It looks like this module only allows you to create one shared store.

I have different states that I need to share between different windows. For instance, Store A needs to be shared between Main and Window 1, while Store B needs to be shared between Main, Window 1, and Window 2.

Would you be open to a pull request which adds the ability to create multiple named stores? Offhand, it might look like this:

```JS
const sharedStoreA = createSharedStore(initialStateA, 'store_a')
const sharedStoreB = createSharedStore(initialStateB, 'store_b')
```

Then we might append this name onto the `INTERNAL_CHANNEL` value so that the IPC listeners would just listen/send on different channels for the different stores. If someone didn't give a storeName value, then we would fall back to the current functionality.

**Comment by zoubingwu** [comment]
hey @jjeff, thanks for your feedback and contribution.

I was wondering is there any particular reason you want to create multiple store?  This repo was inspired by redux's design in some way and it's api makes it easy to put the shared state in a reducer (or rtk's slice).

in your example you could just:

```js
const store = createSharedStore({
  store_a: initialStateA,
  store_b: initialStateB,
})
```

Single store is much simpler, you can also check https://redux.js.org/faq/store-setup#can-or-should-i-create-multiple-stores-can-i-import-my-store-directly-and-use-it-in-components-myself

**Comment by jjeff** [comment]
One of my stores can get VERY large. It basically manages a video library with all of its metadata. That store is shared between the `main` process and a window that displays the library data. The other window is the playback window. It needs to be light and perform very quickly. So I don't want it to be weighed down by updates to my video library file. Your solution doesn't stop my library data from being synced to (and slowing down) my playback window.

**Comment by zoubingwu** [comment]
published in v1.1.0
